packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+from matplotlib.backend_bases import GraphicsContextBase, RendererBase
+
+_capstyle_d = {'projecting' : 'square', 'butt' : 'butt', 'round': 'round'}
+
+class GraphicsContextWasm(GraphicsContextBase):
```

Comment:
Maybe we should rename this to `GraphicsContextHTMLCanvas`?  (Everything, including the existing backend is Wasm, so HTML canvas is probably more specific)

Reply:
Aah yes, I think we should also change `RendererWasm` to `RendererHTMLCanvas` as well then. I will also update the description above.